templates.dist/estate/default.php

```diff
@@ -30,6 +30,53 @@
 
 $dontEcho = array("objekttitel", "objektbeschreibung", "lage", "ausstatt_beschr", "sonstige_angaben", "MPAreaButlerUrlWithAddress", "MPAreaButlerUrlNoAddress");
 
+/*  responsive picture properties
+ *  customizable widths and heights for individual layouts
+ */
+$image_width_xs = 300;
+$image_width_sm = 350;
+$image_width_md = 450;
+$image_width_lg = 250;
+$image_width_xl = 250;
+$image_width_xxl = 250;
+$image_width_xxxl = 250;
```

Comment:
Wie ermittelst du die maximale Breite für ein Bild?
Ich bin hier von einem fluid container ausgegangen, in dem der Inhalt liegt und habe mir die Breiten bei einem dreiteiligen Layout angeschaut. 

Hast du hier vielleicht  Höhe und Breite vertauscht?
Ist das bei detail und similar estate eventuell der selber Grund?

Reply:
Ich hatte das mit der W3PM Neve-Child Instanz getestet (das Ticket weist auch in diese Richtung), aber gerade ist mir klar geworden, dass die Größen hier leider teilweise irrelevant sind, weil jeder Benutzer ein anderes Theme mit unterschiedlichen Inhaltsbreiten verwenden kann. Das führt dazu, dass wir keine „richtigen“ Größen für das Standard-Template liefern können, da die Seiten immer unterschiedlich sein werden.

~~Die Breiten hatte ich manuell anhand der Elementbreite für die jeweiligen Breakpoints getestet:~~

~~Zum Beispiel bei 576px - 768px (sm) ist der Mittelwert des Breakpoints 672px, und dafür hatte ich 350px genommen, weil, wenn 2 Spalten angezeigt werden, die 300px & 300px breit sind, bei 672px dann ein Bild 50px herunterskaliert werden muss, und bei 767px ist es dann 350px breit.~~

Reply:
Genau deswegen habe ich versucht eine sinnvolle größtmögliche Breite bei einem dreiteiligen Layout zu wählen. 
Hast du einen besseren Vorschlag, als dieses Vorgehen? 
Wenn das Styling customized wurde, werden wahrscheinlich eigene Templates verwendet und da können die Breakpoint Größen individuell angepasst werden. 

Ich werde den Commit rückgängig machen.

Reply:
Nein, deine Bildgrößen können dann bleiben. Nur bei xs verwende bitte eine kleinere Größe als 545px. 
Lighthouse Mobile testet mit 412px device width